### PR TITLE
update: added What's new in Kotlin 1.4.30 for KMM

### PR DESCRIPTION
**Pull request by katia-energizer**

### CODE REVIEW

**Review by lis-balbes — APPROVED** [review]

**Review by KaterinaPetrova — APPROVED** [review]

File: topics/whats-new-in-kotlin-for-kmm.md [hunk 1]
```diff
@@ -4,8 +4,25 @@
...
 
+Android Studio will recommend an automatic update to a new Kotlin release. You can also [update manually](https://kotlinlang.org/releases.html#updating-to-a-new-release).
+
 Here you can find a short summary of the features Kotlin provides for developing multiplatform mobile applications.
 
+## Kotlin 1.4.30 for KMM
+
+[Kotlin 1.4.30](https://kotlinlang.org/docs/reference/whatsnew1430.html) introduces a number of improvements that are helpful for KMM:
+
+* **Improved compilation time for iOS Simulator**. Recompiling binaries targeting iOS Simulator after making changes in the code now requires much less time.
```
Comment (p7nov) [inline topics/whats-new-in-kotlin-for-kmm.md:15]: As I see in [this Xcode doc page](https://developer.apple.com/documentation/xcode/running_your_app_in_the_simulator_or_on_a_device) the word `simulator` is capitalized only when it refers to the [_Simulator_ app](https://help.apple.com/simulator/mac/current/#/deve44b57b2a), which is hardly the case for our docs.
In other cases, such as referring to specific platform simulators, it's just `(a/the) simulator`.

Comment (p7nov) [reply topics/whats-new-in-kotlin-for-kmm.md:15]: Also, the sentence is a bit hard to read with three gerunds in a row. 
> Recompiling ... targeting ... after making ...

Comment (katia-energizer) [reply topics/whats-new-in-kotlin-for-kmm.md:15]: Copyeditors corrected capitalization for Simulator and liked three gerunds. 😂 
But let's remove one gerund and capitalization. I google and iOS Simulator is often capitalized. But it refers to the app. I agree.

Comment (katia-energizer) [reply topics/whats-new-in-kotlin-for-kmm.md]: Updated. Please take a look.

**Review by p7nov — APPROVED** [review]

**Pull request merged**